tests/entrypoints/openai/test_response_api_with_harmony.py

```diff
@@ -504,7 +504,14 @@ async def test_web_search(client: OpenAI, model_name: str):
 @pytest.mark.asyncio
 @pytest.mark.parametrize("model_name", [MODEL_NAME])
 async def test_code_interpreter(client: OpenAI, model_name: str):
-    response = await client.responses.create(
+    # Code interpreter needs more time for container init + code execution
```

Comment:
just make the timeout longer for all platforms

Reply:
Done :)